### PR TITLE
feature(ui-ux): update language screen v2.0

### PR DESCRIPTION
**Pull request by G9000**
#### What kind of PR is this?:
/kind feature

#### What this PR does / why we need it:
Update Language screen 2.0

#### Which issue(s) does this PR fixes?:
Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx [hunk 1]
```diff
@@ -2,22 +2,25 @@ import { ThemedScrollView } from '@components/themed'
...
 import { RowLanguageItem } from '../components/RowLanguageItem'
+import { tailwind } from '@tailwind'
 
 export function LanguageSelectionScreen (): JSX.Element {
   const languages = getAppLanguages()
 
   return (
     <ThemedScrollView testID='language_selection_screen'>
       <ThemedSectionTitle
+        style={tailwind('px-8 pt-6 pb-2 text-xs text-gray-500 font-medium')}
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx:13]: screen background looks different from Figma. Need to double check

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx (hunk 1 above)
Comment (G9000) [reply mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx]: check if its correct already :v

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx [hunk 2]
```diff
@@ -2,22 +2,27 @@ import { ThemedScrollView } from '@components/themed'
 import { ThemedSectionTitle } from '@components/themed/ThemedSectionTitle'
 import { getAppLanguages, translate } from '@translations'
 import { RowLanguageItem } from '../components/RowLanguageItem'
+import { tailwind } from '@tailwind'
 
 export function LanguageSelectionScreen (): JSX.Element {
   const languages = getAppLanguages()
 
   return (
     <ThemedScrollView testID='language_selection_screen'>
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx:11]: Should have a different background here. Please refer to the actual figma file for both light and Dark

File: mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx [hunk 3]
```diff
@@ -40,38 +42,45 @@ export function RowLanguageItem ({ languageItem }: { languageItem: AppLanguageIt
...
-      <View>
-        <ThemedText testID='language_option' style={tailwind('font-medium')}>
-          {languageItem.displayName}
-        </ThemedText>
-        <ThemedText
-          testID='language_option_description'
-          dark={tailwind('text-gray-400')}
-          light={tailwind('text-gray-500')}
-          style={tailwind('text-sm')}
+      <View style={tailwind(`flex flex-row mx-5 py-4 items-center justify-between border-b-0.5 ${isLight ? 'border-mono-light-v2-300' : 'border-mono-dark-v2-300'}`, { 'border-b-0': lastItem })}>
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:56]: I think we're losing some functionality here. Let's keep it as what we have now on the app.

Display Language (Translated language on the select label).

File: shared/translations/index.ts [hunk 4]
```diff
@@ -94,8 +94,8 @@ export function encodeScope (text: string): string {
 export enum AppLanguage {
   English = 'English',
   German = 'German',
-  ChineseSimplified = 'Chinese (Simplified)',
```
Comment (thedoublejay) [inline shared/translations/index.ts:97]: Need translations for these

File: mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx (hunk 3 above)
Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:56]: Screenshot_20220709-112214_DeFiChain Wallet

Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:56]: Example, the text below is translated. For the new layout, let's translate also the one inside the ( )

File: mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx [hunk 5]
```diff
@@ -40,38 +42,45 @@ export function RowLanguageItem ({ languageItem }: { languageItem: AppLanguageIt
...
+        <View style={tailwind('flex flex-row items-center')}>
+          <ThemedText testID='language_option' style={tailwind('font-normal-v2')}>
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:58]: Think this one should be font-xs

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx (hunk 2 above)
Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx:11]: You can replace it with this component <ThemedScrollViewV2, automatically adds the default BG

File: mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx (hunk 5 above)
Comment (G9000) [reply mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:58]: This is probably an old commit. Resolve in newer commit

Comment (G9000) [reply mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx:58]: I assume you mean text-xs. But anyway I think the above is an old commit and should be fixed on a newer commit.
text-sm is 14px right which is the same as the figma. text-xs is 12px

File: mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx (hunk 2 above)
Comment (G9000) [reply mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx:11]: Done haha

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**